pulsar/json_schemas/channel.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://asyncapi.com/bindings/pulsar/channel.json",
+    "title": "Channel Schema",
+    "description": "This object contains information about the channel representation in Pulsar.",
+    "type": "object",
+    "additionalProperties": false,
+    "patternProperties": {
+      "^x-[\\w\\d\\.\\-\\_]+$": {
+        "$ref": "https://raw.githubusercontent.com/asyncapi/spec-json-schemas/v2.14.0/schemas/2.4.0.json#/definitions/specificationExtension"
+      }
+    },
+    "properties": {
+      "namespace": {
+        "type": "string",
+        "description": "The namespace, the channel is associated with. If omitted, 'public' MUST be assumed."
+      },
+      "persistence": {
```

Comment:
I saw another binding that used strings for this,which is why i went with strong, but yes, as pulsar only has 2 it should be fine.

The argument for string could be, that it is easier to infer the topic string when connecting. Instead of having to know the vendor specifics that persistence means "persistent", but rather a topic looks like 
`{persistence}/{tenant}/{namspace}/{channel}` if you know what I mean. Which i personally think is a bit closer to what we want out of the binding.